docker-compose.yml

```diff
@@ -155,6 +155,43 @@ services:
       - gitbase
       - bblfsh-web
 
+  sourced-ui-celery:
+    image: srcd/sourced-ui:latest
+    restart: unless-stopped
+    environment:
+      SYNC_MODE: ${GITBASE_SIVA}
```

Comment:
Maybe we could use extension fields to define the list of common env vars only once. WDYT? https://docs.docker.com/compose/compose-file/#extension-fields

Reply:
I see it requires `3.4` file format. In the documentation, they say it requires docker 17.09.0+ which is fine. But I don't see any mention if there are restrictions on docker-compose itself. 
I'll dig into it.
Anyway thanks for the link, I wasn't aware of YAML anchors support by newer docker-compose.

Reply:
I missed this, this is cool! 🚀